shared/constants/network.ts

```diff
@@ -49,25 +44,21 @@ type MoonPayNetworkAbbreviation = 'bsc' | 'cchain' | 'polygon';
 type MoonPayChainSettings = {
   /**
    * What should the default onramp currency be, for example 'eth' on 'mainnet'
-   * This type matches a single LowercaseCurrencySymbol or a
-   * LowercaseCurrencySymbol and a MoonPayNetworkAbbreviation joined by a '_'.
+   * This type matches a single SupportedCurrencySymbol or a
+   * SupportedCurrencySymbol and a MoonPayNetworkAbbreviation joined by a '_'.
    */
   defaultCurrencyCode:
-    | LowercaseCurrencySymbol
-    | `${LowercaseCurrencySymbol}_${MoonPayNetworkAbbreviation}`;
+    | SupportedCurrencySymbol
+    | `${SupportedCurrencySymbol}_${MoonPayNetworkAbbreviation}`;
   /**
    * We must also configure all possible onramp currencies we wish to support.
-   * This type matches 1 to 3 LowercaseCurrencySymbols, joined by ','. It also
-   * matches 1 or 2 LowercaseCurrencySymbols with a
+   * This type matches 1 to 3 SupportedCurrencySymbol, joined by ','. It also
+   * matches 1 or 2 SupportedCurrencySymbol with a
    * MoonPayNetworkAbbreviation joined by a '_', and concatenated with ','.
    */
   showOnlyCurrencies:
-    | `${LowercaseCurrencySymbol}`
-    | `${LowercaseCurrencySymbol},${LowercaseCurrencySymbol}`
-    | `${LowercaseCurrencySymbol},${LowercaseCurrencySymbol},${LowercaseCurrencySymbol}`
-    | `${LowercaseCurrencySymbol},${LowercaseCurrencySymbol},${LowercaseCurrencySymbol},${LowercaseCurrencySymbol}`
-    | `${LowercaseCurrencySymbol}_${MoonPayNetworkAbbreviation}`
-    | `${LowercaseCurrencySymbol}_${MoonPayNetworkAbbreviation},${LowercaseCurrencySymbol}_${MoonPayNetworkAbbreviation}`;
+    | SupportedCurrencySymbol[]
+    | `${SupportedCurrencySymbol}_${MoonPayNetworkAbbreviation}`[];
```

Comment:
Oh i see its an array now instead of a joined list. Could you update the doc block above to remove the '1 to 3' joined by ',' and '1 or 2' 'and concatenated with ','' portions? The array is much better and the reason why this is passing TSC now 👏🏻

Reply:
thanks for taking a look @brad-decker ! good catch btw, just updated the doc block that was describing the previous typing.